forms-web-app/tests/unit/controllers/appellant-submission/site-access-safety.test.js

```diff
@@ -5,18 +5,29 @@ const logger = require('../../../../src/lib/logger');
 const { getNextUncompletedTask } = require('../../../../src/services/task.service');
 const { mockReq, mockRes } = require('../../mocks');
 const { APPEAL_DOCUMENT } = require('../../../../src/lib/empty-appeal');
+const { getTaskStatus } = require('../../../../src/services/task.service');
 
 jest.mock('../../../../src/lib/appeals-api-wrapper');
 jest.mock('../../../../src/services/task.service');
 jest.mock('../../../../src/lib/logger');
 
-const req = mockReq();
-const res = mockRes();
-
 const sectionName = 'appealSiteSection';
 const taskName = 'healthAndSafety';
 
 describe('controllers/appellant-submission/site-access-safety', () => {
+  let req;
+  let res;
+  let appeal;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+
+    ({ empty: appeal } = APPEAL_DOCUMENT);
```

Comment:
TIL: reassigning via destructuring ... :boom: 

![tenor](https://user-images.githubusercontent.com/6329541/103554086-49862d80-4ea6-11eb-944d-aeeab3efbabc.gif)

Reply:
lol - starting the new year with enthusiasm?